Hash_map/test/Hash_map/Hash.cpp

```diff
@@ -36,14 +36,14 @@ template <typename P>
 void
 fct(const P& )
 {
-  typedef typename boost::graph_traits<P>::vertex_descriptor vertex_descriptor;
+  typedef typename boost::graph_traits<P>::edge_descriptor edge_descriptor;
 
-  std::map<vertex_descriptor,int> M;
-  vertex_descriptor vd;
-  M.find(vd);
+  std::map<edge_descriptor,int> M;
+  edge_descriptor ed;
+  M.find(ed);
 
-  boost::unordered_map<vertex_descriptor, int> U;
-  U[vd] = 12;
+  boost::unordered_map<edge_descriptor, int> U;
+  U[ed] = 12;
 }
```

Comment:
It seems the test was previously testing the hash function for `boost::graph_traits<P>::vertex_descriptor`, and now it only tests the hash function for `boost::graph_traits<P>::edge_descriptor`, and no longer the one for vertices. In my opinion it should test both.